### ISSUE TITLE
Lens needs replacing on Tony

### ISSUE THREAD

**Issue by magman2112** [DoESLiverpool/somebody-should#1917]
The cutting on Tony has been having some issues lately. I inspected the lens on Maker Day and found the lens to be badly marked with an internal flaw. 

I managed to clean the lens to a degree, but the flaw is still present and will cause the lens to mark up again quickly, so a replacement is due.

This was previously an issue in #1749, where we indentified a suitable lens from Amazon. On checking the amazon site though, delivery for the original item is 2-3weeks. So I managed to find an alternate supplier on Amazon which is next day delivery.

For reference, the lens is 19mm diameter with a focal length of 63.5mm (or 2 1/2 inches).

https://www.amazon.co.uk/TEN-HIGH-19-05mm-Focal-Laser-Cutting/dp/B01EQQJ202/

If one of these could be purchased soon, we can then confirm this fits and works correctly, then likely order a second one to keep one on hand as a spare.

@DoESLiverpool/laser-maintenance

**Comment by DoESsean** [comment]
Should I go ahead and buy this lens then?

**Comment by johnmckerrell** [comment]
Yes--Sent from my mobile phone hence brevity and errorsOn 11 Jun 2024, at 11:17, Seán Gleeson ***@***.***> wrote:﻿
Should I go ahead and buy this lens then?

**Comment by DoESsean** [comment]
Great, it's been ordered today. is supposed to come tomorrow.

**Comment by magman2112** [comment]
Thanks Sean, that means I can test it on Thursday when I am next in.

**Comment by DoESsean** [comment]
It's arrived, I'll leave it on top of my laptop for tonight.

**Comment by magman2112** [comment]
The lens was fitted last Maker Day.

Tony appears to be cutting well, but I will do some more testing before seeing if we want to order another lens to hold as a spare.

**Comment by magman2112** [comment]
Closing this issue as another lens has now been fitted with a more recent issue.
